index.bs

```diff
@@ -701,19 +712,19 @@ Reflective modules are always already instantiated.
   <li>re-export (immutable): <code>{ module: m, import: "foo" }</code>
 </ul>
 
-<pre emu-alg>
+<emu-alg aoid="ParseExportsDescriptors">
 1. // TODO: spec me
-</pre>
+</emu-alg>
 
 <h4 id="create-module-mutator">CreateModuleMutator(module)</h4>
 
-<pre emu-alg>
+<emu-alg aoid="CreateModuleMutator">
 1. // TODO: spec me
-</pre>
+</emu-alg>
 
 <h4 id="reflect-module">Reflect.Module(descriptors[, executor, evaluate])</h4>
 
-<pre emu-alg>
+<emu-alg aoid="Reflect.Module">
```

Comment:
I've never seen a precedent for "Reflective Module.GetExportNames" in the spec so I'm not sure. It seems like an abstract op to me so maybe aoid is OK.